mergify_engine/web/api/security.py

```diff
@@ -100,11 +100,8 @@ async def get_repository_context(
     repository: github_types.GitHubRepositoryName = fastapi.Path(  # noqa: B008
         ..., description="The name of the repository"
     ),
-    redis_cache: utils.RedisCache = fastapi.Depends(  # noqa: B008
-        redis.get_redis_cache
-    ),
-    redis_queue: utils.RedisQueue = fastapi.Depends(  # noqa: B008
-        redis.get_redis_queue
+    redis_links: redis_utils.RedisQueue = fastapi.Depends(  # noqa: B008
```

Comment:
```suggestion
    redis_links: redis_utils.RedisLinks = fastapi.Depends(  # noqa: B008
```

Reply:
Hum, wierd that mypy didn't cached that one...